Deepgram/Clients/Agent/v2/Websocket/Client.cs

```diff
@@ -51,6 +52,10 @@ public Client(string? apiKey = null, IDeepgramClientOptions? options = null) : b
     public async Task<bool> Connect(SettingsConfigurationSchema options, CancellationTokenSource? cancelToken = null, Dictionary<string, string>? addons = null,
         Dictionary<string, string>? headers = null)
     {
+        if (!options.Agent.Listen.Model.StartsWith("nova-3") && options.Agent.Listen.Keyterms?.Count > 0)
+        {
+            throw new DeepgramException("Keyterms is only supported in Nova 3 models.");
```

Comment:
I was debating if we needed this exception in the SDKs as our API enforcing it.

Reply:
Discussed in Slack - moot point, we're fine either way as these SDKs will soon be approached differently.